_episodes/09-The Push.md

```diff
@@ -20,13 +20,15 @@ Push events happen **three** times a year. They start on the third Thursday of t
 
 With the increased number of Nodes joining the Pushes, we are announcing the schedule for the next year. Please prepare in advance and mark your calendars.
 
-Push schedule through 2023:
+Example Push schedule through 2023:
```

Comment:
should this be updated to 2024?

Reply:
@CaitlinBate please add the 2024 and future dates